feature/dashboard/src/main/java/com/simprints/feature/dashboard/settings/syncinfo/SyncInfoViewModel.kt

```diff
@@ -140,8 +145,16 @@ internal class SyncInfoViewModel @Inject constructor(
                 eventSyncButtonClickFlow.emit(Unit)
             }
             syncOrchestrator.stopEventSync()
-            val isDownSyncAllowed =
-                !isPreLogoutUpSync && configManager.getProject(authStore.signedInProjectId).state == ProjectState.RUNNING
+            val projectState = try {
+                configManager.getProject(authStore.signedInProjectId).state
+            } catch (_: Exception) {
+                // When the device is compromised the project data will be deleted and
+                // attempting to access project state with result in exception.
+                // For user it is essentially the same as project ending.
```

Comment:
Not to nitpick but we do have ENDING and ENDED project states so there should not be confusion which one is meant here.